samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test rx_lna_gain 2
+              wifi_radio_test rx_bb_gain 10
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_cap 1
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              E940F0
+              E8EFE7
+              F3FF29
+              03CF27
+              0F7FD3
+              1020D4
+              054194
+              F5519E
+              E990F3
+              E8FFF4
+              F3AF32
+              038F20
+              0F5FC9
+              1040CD
+              05D191
+              F631A8
+              EA1102
+              E88FFF
+              F2EF36
+              029F1D
+              0EDFBE
+              1090C0
+              06718F
+              F6A1AB
+              EA410B
+              E8200B
+              F1FF39
+              01DF1A
+              0E9FBB
+              10B0B9
+              06D189
+              F711AE
+              EA6113
+              E83015
+              F1BF41
+              017F17
+              0E4FAC
+              10B0AB
+              076185
+              F7A1AD
+              EAA11A
+              E8101D
+              F18F45
+              00EF15
+              0E0FA7
+              10F0A3
+              07D180
+              F851B4
+              EB5120
+              E83025
+              F0FF4D
+              00AF15
+              0DDFA1
+              1110A2
+              08217A
+              F871B0
+              EB3126
+              E79027
+              F07F4B
+              007F12
+              0DAF9B
+              115097
+              08A178
+
+
+         .. note::
+
+            * This test requires a DUT to be connected with VSG using cabled setup. VSG should be continuously transmitting
+            * a tone or packets. Set signal power level between -35dBm to -45dBm to get non saturated samples. In the above
+            * example capture is taken for 64 samples. Each complex sample is of 24 bits.
+            * The capture shown above is just an example. The captured samples will vary from run to run.
+
+
+
+         * To capture AGC adjusted baseband samples after WLAN packet detection, with the following configuration:
+
+           * Channel: 144
+           * Capture length: 64
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_dyn_pkt_cap
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              83E6824F
```

Comment:
These values are not of interest since they will vary from run to run. Can we use some form of <- snip -> notation to only show a few lines ?

Reply:
Done.